src/grafana_dashboards/postgresql-metrics.json

```diff
@@ -1995,7 +2006,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "pg_stat_database_tup_fetched{datname=~\"$datname\", instance=~\"$instance\"} != 0",
+          "expr": "irate(pg_stat_database_tup_fetched{datname=~\"$datname\", instance=~\"$instance\"}[5m])",
```

Comment:
for the rate-type metrics, we use `irate`. For count-type, we use the plain metric.